### PR TITLE
III-4095 - Add sidebar link filminvoer

### PR DESCRIPTION
**Pull request by simon-debruijn**
### Added

- path to `/manage/movies/create`
- Page with page title
- redirect
- menu item for movies

### Changed

- transformed multiple components to typescript

### Fixed

- badge in mobile view

---

Ticket: https://jira.uitdatabank.be/browse/III-4095

Screenshot:
Schermafbeelding 2021-07-08 om 15 33 21

### CODE REVIEW

File: src/components/layouts/Sidebar.tsx [hunk 1]
```diff
@@ -32,8 +34,40 @@ const getValueForMenuItem = getValueFromTheme('menuItem');
 const getValueForSidebar = getValueFromTheme('sidebar');
 const getValueForMenu = getValueFromTheme('menu');
 
+const PermissionTypes = {
+  AANBOD_BEWERKEN: 'AANBOD_BEWERKEN',
```
Comment (brampauwelyn) [inline src/components/layouts/Sidebar.tsx:38]: Did you consider using english names for the keys?

File: src/redirects.js [hunk 2]
```diff
@@ -33,7 +33,7 @@ const createDashboardRedirects = (environment) => {
   ];
 };
 
-const getRedirects = (environment) => [
+const getRedirects = (environment, language) => [
```
Comment (brampauwelyn) [inline src/redirects.js:36]: Would it be an idea to add a default value?

**Review by brampauwelyn — APPROVED** [review]

File: src/redirects.js (hunk 2 above)
Comment (brampauwelyn) [reply src/redirects.js:36]: + no types here? 😃

File: src/components/layouts/Sidebar.tsx (hunk 1 above)
Comment (simon-debruijn) [reply src/components/layouts/Sidebar.tsx:38]: I would also prefer english names, but here we use the same ones comming from the udb3-backend also in dutch

File: src/redirects.js (hunk 2 above)
Comment (simon-debruijn) [reply src/redirects.js:36]: Haha I'l typescriptify it for you

And good idea

**Pull request merged**